lib/routes/sicau/jk.ts

```diff
@@ -0,0 +1,243 @@
+import { DataItem, Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import cache from '@/utils/cache';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/jk/:gid/:typeId/:sortType/:token',
+    categories: ['university'],
+    example: '/sicau/jk/0/0/2/8d95466cf63e537292b303cb92b5958c',
+    parameters: {
+        gid: '活动所属组织ID，见下表',
+        typeId: '活动类别ID，见下表',
+        sortType: '排序方式，见下表',
+        token: '访问令牌，可通过示例中的令牌直接访问（会过期）',
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '二课活动',
+    maintainers: ['hualiong'],
+    url: 'jk.sicau.edu.cn',
+    description: `
+:::tip
+**本校学生**可以直接 POST \`https://jk.sicau.edu.cn//user/login/v1.0.0/snoLogin\` 从返回结果中的 \`token\` 字段拿到个人令牌
```

Comment:
I think it should be `https://jk.sicau.edu.cn/user/login/v1.0.0/snoLogin` (without duplicated slash)

Reply:
Oh, you're right...